diktat-gradle-plugin/build.gradle.kts

```diff
@@ -30,6 +30,8 @@ dependencies {
         exclude("com.pinterest.ktlint", "ktlint-ruleset-standard")
     }
     implementation("com.pinterest.ktlint:ktlint-reporter-plain:$ktlintVersion")
+    implementation("com.pinterest.ktlint:ktlint-reporter-json:$ktlintVersion")
+    implementation("com.pinterest.ktlint:ktlint-reporter-checkstyle:$ktlintVersion")
```

Comment:
Are these three dependencies needed? I think we are not using any classes of reporters in code of diktat-gradle-plugin itself?

Reply:
Yeah, forgot to delete it